### ISSUE TITLE
RGB color aren't matched

### ISSUE THREAD

**Issue by dwido** [arshad/Sketch-Sass-Colors#7]
Hi,

We have this set of variables in the plugin configuration:
$obbtn-color-background-add-button: rgb(135, 135, 170)
$color-brand: rgb(21, 137, 238)
$color-brand-dark: rgb(0, 112, 210)

And In Sketch:
<img width="217" alt="screen shot 2017-06-18 at 12 12 19" src="https://user-images.githubusercontent.com/154235/27259612-73baebf2-541f-11e7-9af8-074fe4111265.png">

The plugin doesn't match rgb, is that by design?

**Comment by arshad** [comment]
@dwido Yes. Only hex color values are supported. 

But we could add rgb. I'll mark this ticket as enhancement and get back to it this week.

**Comment by damianof** [comment]
What happened? This is so useful, are you planning to still maintain this code?

**Comment by arshad** [comment]
@damianof Yes. I've been busy with work. But I can look at this again next week. Hopefully.
